src/models/Observation.js

```diff
@@ -60,8 +61,20 @@ class Observation extends Realm.Object {
     };
   }
 
+  static async import( obs ) {
+    return {
+      ...obs,
+      captive_flag: false,
+      geoprivacy: "open",
+      owners_identification_from_vision: false,
+      observed_on_string: createObservedOnStringForUpload( ),
+      quality_grade: "needs_id",
+      uuid: uuid.v4( )
+    };
+  }
```

Comment:
You've made a new model method that makes a new observation without any location information... but that's not really *importing* an observation, since you're not getting info from elsewhere, you're just making an obs with different attributes.

My request in the issue was to completely remove any location fetching from the model, so IMO that doesn't require a different method like this, it just requires taking those lines out of the `new()` method and performing location fetching from elsewhere, like an effect in the component.